mirror-soul/PUSH_NOTIFICATION_ARCHITECTURE.md

```
@@ -1,253 +1,73 @@
-  }
+백엔드(`mirror-soul-back`)는 `firebase-admin` SDK로 직접 `FirebaseMessaging.sendEachForMulticast`를 호출하므로, FE는 반드시 **`getDevicePushTokenAsync()`**를 써야 한다. Expo 토큰을 보내면 백엔드가 FCM에 전달할 수 없는 문자열을 받게 된다.
 
-  // 5. Expo Push Token 발급 (iOS의 APNs, Android의 FCM을 Expo가 통합 처리)
-  const token = (await Notifications.getExpoPushTokenAsync({
-    projectId: 'YOUR_EAS_PROJECT_ID', // app.json의 extra.eas.projectId
-  })).data;
+## 2. 전체 흐름
 
-  return token; // "ExponentPushToken[xxxxxxxxxxxxxxxxxxxxxx]"
-}
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)